<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1458</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;Times New Roman&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="regexInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>421</width>
      <height>131</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>如果正则表达式格式出错, 会出现异常</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color: rgb(233, 233, 233);
font: 13pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>输入正则表达式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analysisButton">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>80</y>
      <width>103</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color: rgb(170, 170, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>analyse</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>1271</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="nfa">
     <attribute name="title">
      <string>nfa</string>
     </attribute>
     <widget class="QTableWidget" name="nfaTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>551</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>671</width>
        <height>481</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>669</width>
         <height>479</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="nfaImg">
          <property name="text">
           <string>image</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="dfa">
     <attribute name="title">
      <string>dfa</string>
     </attribute>
     <widget class="QTableWidget" name="dfaTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>551</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>10</y>
        <width>581</width>
        <height>481</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>579</width>
         <height>479</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="dfaImg">
          <property name="text">
           <string>image</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="miniDfa">
     <attribute name="title">
      <string>miniDfa</string>
     </attribute>
     <widget class="QTableWidget" name="miniTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>551</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>571</width>
        <height>481</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>569</width>
         <height>479</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="minidfaImg">
          <property name="text">
           <string>image</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="code">
     <attribute name="title">
      <string>code</string>
     </attribute>
     <widget class="QTextEdit" name="codeText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>1121</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>1150</x>
        <y>250</y>
        <width>103</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;
background-color: rgb(170, 170, 255);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
